packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1283,19 +1283,14 @@ class BuildOwner {
         int dirtyCount = _dirtyElements.length;
         int index = 0;
         while (index < dirtyCount) {
-          assert(() {
```

Comment:
Yeah, it was too noisy here. Now it only prints when you rebuild something active and dirty.